features/admin.applications.v1/components/forms/advanced-configurations-form.tsx

```diff
@@ -198,125 +232,171 @@ export const AdvancedConfigurationsForm: FunctionComponent<AdvancedConfiguration
-                                ".skipConsentLogout.label")
-                        }
-                        required={ false }
-                        value={ config?.skipLogoutConsent ? [ "skipLogoutConsent" ] : [] }
-                        readOnly={ readOnly }
-                        data-testid={ `${ testId }-skip-logout-consent-checkbox` }
-                        data-componentid={ `${ testId }-skip-logout-consent-checkbox` }
-                        hint={ t("applications:forms.advancedConfig.fields.skipConsentLogout" +
-                            ".hint"
-                        ) }
-                    />
-                )
-            }
-            <Field.CheckboxLegacy
-                ariaLabel="Return authenticated IDP list"
-                name="returnAuthenticatedIdpList"
-                label={ t("applications:forms.advancedConfig.fields." +
-                    "returnAuthenticatedIdpList.label"
-                ) }
-                required={ false }
-                value={ config?.returnAuthenticatedIdpList ? [ "returnAuthenticatedIdpList" ] : [] }
-                readOnly={ readOnly }
-                data-testid={ `${ testId }-return-authenticated-idp-list-checkbox` }
-                data-componentid={ `${ testId }-return-authenticated-idp-list-checkbox` }
-                hidden={ !applicationConfig.advancedConfigurations.showReturnAuthenticatedIdPs }
-                hint={ t("applications:forms.advancedConfig.fields" +
-                    ".returnAuthenticatedIdpList.hint"
-                ) }
-            />
-            <Field.CheckboxLegacy
-                ariaLabel="Enable authorization"
-                name="enableAuthorization"
-                label={ t("applications:forms.advancedConfig.fields." +
-                    "enableAuthorization.label"
-                ) }
-                required={ false }
-                value={ config?.enableAuthorization ? [ "enableAuthorization" ] : [] }
-                readOnly={ readOnly }
-                data-testid={ `${ testId }-enable-authorization-checkbox` }
-                data-componentid={ `${ testId }-enable-authorization-checkbox` }
-                hidden={
-                    !applicationConfig.advancedConfigurations.showEnableAuthorization
-                    || !UIConfig?.isXacmlConnectorEnabled }
-                hint={ t("applications:forms.advancedConfig.fields.enableAuthorization.hint") }
-            />
-            {
-                (
-                    isApplicationNativeAuthenticationEnabled &&
-                    (template?.id === ApplicationManagementConstants.CUSTOM_APPLICATION ||
-                    template?.id === ApplicationManagementConstants.CUSTOM_APPLICATION_OIDC ||
-                    template?.id === ApplicationManagementConstants.MOBILE ||
-                    template?.id === ApplicationManagementConstants.TEMPLATE_IDS.get("oidcWeb"))
-                ) && (
-                    <>
+        <>
+            <Form
```

Comment:
Is this indentation change intentional?

Reply:
This indentation was due to the newly added ConfirmationModal